packages/firestore/test/unit/remote/serializer.helper.ts

```diff
@@ -661,66 +660,108 @@ export function serializerTest(
         verifyMutation(mutation, proto);
       });
 
-      it('TransformMutation (ServerTimestamp transform)', () => {
-        const mutation = transformMutation('baz/quux', {
+      it('ServerTimestamp transform', () => {
+        const mutation = patchMutation('baz/quux', {
           a: FieldValue.serverTimestamp(),
           'bar.baz': FieldValue.serverTimestamp()
         });
         const proto = {
-          transform: {
-            document: toName(s, mutation.key),
-            fieldTransforms: [
-              { fieldPath: 'a', setToServerValue: 'REQUEST_TIME' },
-              { fieldPath: 'bar.baz', setToServerValue: 'REQUEST_TIME' }
-            ]
-          },
+          update: toMutationDocument(s, mutation.key, mutation.data),
+          updateMask: toDocumentMask(mutation.fieldMask),
```

Comment:
You should add a comment to the `fieldMask` field that states that the mask does not include document transforms.

Reply:
https://github.com/firebase/firebase-js-sdk/blob/80d19e2f62ef1e43d7ea059415d9b93804deb025/packages/firestore/src/api/user_data_reader.ts#L102